packages/live-preview-sdk/src/inspectorMode/types.ts

```diff
@@ -1,14 +1,21 @@
 import type { SetRequired } from 'type-fest';
 
-export type InspectorModeTags = {
+export type InspectorModeEntryTags = {
   [InspectorModeDataAttributes.ENTRY_ID]: string;
   [InspectorModeDataAttributes.FIELD_ID]: string;
   [InspectorModeDataAttributes.LOCALE]?: string;
-} | null;
+};
+export type InspectorModeAssetTags = {
+  [InspectorModeDataAttributes.ASSET_ID]: string;
+  [InspectorModeDataAttributes.FIELD_ID]: string;
+  [InspectorModeDataAttributes.LOCALE]?: string;
+};
+export type InspectorModeTags = InspectorModeEntryTags | InspectorModeAssetTags | null;
 
 export const enum InspectorModeDataAttributes {
   FIELD_ID = 'data-contentful-field-id',
   ENTRY_ID = 'data-contentful-entry-id',
+  ASSET_ID = 'data-contentful-asset-id',
```

Comment:
What do you think?

Reply:
We can also use this library to create assemble / disassemble them: https://github.com/contentful/resource-names-stack/tree/main/packages/resource-names

Reply:
In the discussion we mentioned that this would need spaceId and environmentId as required param, therefore I'm not sure that we wanted to go into that direction.
If we want to go into that way we should ship it with the canary branch as a breaking change?